pro/src/components/IndividualOfferForm/OfferLocation/OfferLocation.tsx

```diff
@@ -61,9 +61,12 @@ export const OfferLocation = ({ venue }: OfferLocationProps): JSX.Element => {
     }
   }
 
-  const venueLabel = `${
-    venue?.publicName ? `${venue.publicName} – ` : ''
-  }« Adresse de l'OA »`
+  const venueAddress = venue?.address
+    ? `${venue.address.street} ${venue.address.postalCode} ${venue.address.city}`
+    : null
+  const venueNameAddress = [venue?.publicName, venueAddress]
+    .filter(Boolean)
+    .join(' – ')
```

Comment:
L'adresse est censée avoir un label, et il existe `computeAddressDisplayName` qui fait ce qu'il faut.

Ensuite, est-ce qu'il faut quand même afficher le publicName s'il n'y a pas d'adresse ?
```suggestion
  const venueNameAddress = venue?.address
    ? computeAddressDisplayName(venue.address)
    : venue?.publicName
```